Controller.php

```diff
@@ -93,25 +167,90 @@ public function gettingStarted()
 
     public function manageTags()
     {
-        return $this->renderManageContainerTemplate('manageTags');
+        $tagsHelpText = $this->renderTemplate('helpContent', [
+            'subHeading' => Piwik::translate('TagManager_ManageTagsHelp1'),
+            'paragraphs' => [
+                Piwik::translate('TagManager_ManageTagsHelp2'),
+                Piwik::translate('TagManager_ManageTagsHelp3', [
+                    '<a href="https://matomo.org/guide/tag-manager/tags/" rel="noreferrer noopener" target="_blank">',
+                    '</a>',
+                    '<a href="https://matomo.org/guide/tag-manager/getting-started-with-tag-manager/" rel="noreferrer noopener" target="_blank">',
+                    '</a>',
+                    '<a href="https://matomo.org/tag-manager-training/" rel="noreferrer noopener" target="_blank">',
```

Comment:
Should we link this to - https://matomo.org/tag-manager-training/#mtm-tags to make it more specific ?

Reply:
From this context, that would make sense, but I pulled the link directly from the Google Doc detailing the changes. Also, the translated string that contains the anchor is 'and watch the [training video series](https://matomo.org/tag-manager-training/).', which seems more general and mentions the series and not the specific video.